go/chat/storage/inbox.go

```diff
@@ -254,6 +254,19 @@ func (i *Inbox) Merge(ctx context.Context, vers chat1.InboxVers, convsIn []chat1
 	return nil
 }
 
+func (i *Inbox) supersedersNotEmpty(ctx context.Context, superseders []chat1.ConversationMetadata, convs []chat1.Conversation) bool {
+	for _, superseder := range superseders {
+		for _, conv := range convs {
+			if superseder.ConversationID.Eq(conv.Metadata.ConversationID) {
+				if len(conv.MaxMsgs) > 0 {
```

Comment:
The reason is that we always have at least 1 message in every conversation (unless it is a broken conversation). It is of type `TLFNAME`. The reason for that is so we can sign into the message stream the name of the TLF it is associated with at the time we create the conversation. So because of that, this length check shoudl always succeed.